admin/class-yoast-network-admin.php

```diff
@@ -161,32 +161,22 @@ public function settings_fields( $option_group ) {
 	}
 
 	/**
-	 * Prints the form for restoring a site with its default settings.
+	 * Enqueues network admin assets.
 	 *
 	 * @return void
 	 */
-	public function print_restore_form() {
```

Comment:
Shouldn't we deprecated this because it's a public method?

Reply:
This method was introduced in a previous PR and was never part of any release, so we can just remove it IMO.